Updated README.md (#13)
Developer-DAO/developer-dao: README.md

    <li><a href="#what-is-this-repo">What is this repo?</a></li>
    <li><a href="#developing">Developing</a></li>
    <li><a href="#contributing">Contributing</a></li>
    <li><a href="#authors">Authors</a></li>
</ul>

## What is a DAO

DAO stands for **D**ecentalized **A**utonomous **O**rganisation. That is a group of people with shared goals/mission that are governed by an agreed set of rules coded into a smart contract(s) on a blockchain.

If you want to learn more about DAOs, check out Cooper Turley's blog at https://alias.co/coopahtroopa/essays, mainly ["The Rise of Micro-Economies"](https://coopahtroopa.mirror.xyz/gWY6Kfebs9wHdfoZZswfiLTBVzfKiyFaIwNf2q8JpgI).

## What is DeveloperDAO

The project was started as an experiment by [Nader Dabit](https://twitter.com/dabit3) to see what can be achieved when we bring together talented and like-minded people who are interested in learning about and building on Web3.

As stated on : "Developers around the world are tired of working and contributing their time and effort to enrich the top 1%. Join the movement that is community owned, building the future from the bottom up."

## How do I join?

> The DAO operates inside our [discord sever](https://twitter.com/dabit3). To gain access to this server you need to own a [Devs for Revolution](https://opensea.io/collection/devs-for-revolution) NFT. The contract for purchasing one [can be found on etherscan.io](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b). If you're not sure how to purchase an NFT, [check out this how-to article](https://jonkuperman.com/how-to-join-developer-dao/) by @jkup. 

Author: Good spot. Fixed in latest push

<!-- LOGO -->

<p align="center">
    <a href="https://developerdao.com">
    <img src="/src/images/ddao_logo.jpeg" alt="logo" width="80" height="80"/>
    </a>
    <h2 align="center">DeveloperDAO</h2>
    <p align="center">
    A community of builders who believe in collective ownership of the internet.
    </p>
</p>

> [![Chat on Discord](https://img.shields.io/badge/chat-Discord-7289DA?logo=discord)](https://discord.com/invite/ZHZup79J) [![Follow on Twitter](https://img.shields.io/badge/social-Twitter-1DA1F2?logo=twitter)](https://twitter.com/developer_dao) [![See on Etherscan](https://img.shields.io/badge/MintNFT-Ethereum-3C3C3D?logo=ethereum)](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b#writeContract)

Reviewer: I wonder if we have enough members yet to qualify for a custom Discord invite URL. I'm not even sure who to ask!

Reviewer: ```suggestion
[![Chat on Discord](https://img.shields.io/badge/chat-Discord-7289DA?logo=discord)][Discord] [![Follow on Twitter](https://img.shields.io/badge/social-Twitter-1DA1F2?logo=twitter)](https://twitter.com/developer_dao) [![See on Etherscan](https://img.shields.io/badge/MintNFT-Ethereum-3C3C3D?logo=ethereum)](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b#writeContract)
```

Reviewer 2: > I wonder if we have enough members yet to qualify for a custom Discord invite URL. I'm not even sure who to ask!

This might help: https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs

Author: Looks like we need sever boosting so no can do atm :(

<!-- Table of Contents -->

<summary><h2 style="display: inline-block">Table of Contents</h2></summary>
<ul>
    <li><a href="#what-is-a-dao">What is a DAO</a></li>
    <li><a href="#what-is-developerDAO">What is  DeveloperDAO</a></li>
    <li><a href="#how-do-i-join">How do I join?</a></li>
    <li><a href="#what-is-this-repo">What is this repo?</a></li>
    <li><a href="#developing">Developing</a></li>
    <li><a href="#contributing">Contributing</a></li>
    <li><a href="#authors">Authors</a></li>
</ul>

## What is a DAO

> DAO stands for Decentalized autonomous organisation. That is a group of people with shared goals/mission that are governed by an agreed set of rules coded into a smart contract(s) on a blockchain.

Reviewer: ```suggestion
DAO stands for **D**ecentalized **A**utonomous **O**rganisation. That is a group of people with shared goals/mission that are governed by an agreed set of rules coded into a smart contract(s) on a blockchain.
```

> If you want to learn more about DAOs, check out Cooper Turley's blog at https://alias.co/coopahtroopa/essays, mainly "The Rise of Micro-Economies"
> 
> https://coopahtroopa.mirror.xyz/gWY6Kfebs9wHdfoZZswfiLTBVzfKiyFaIwNf2q8JpgI

Reviewer: ```suggestion
If you want to learn more about DAOs, check out Cooper Turley's blog at https://alias.co/coopahtroopa/essays, mainly ["The Rise of Micro-Economies"](https://coopahtroopa.mirror.xyz/gWY6Kfebs9wHdfoZZswfiLTBVzfKiyFaIwNf2q8JpgI).
```

## What is DeveloperDAO

> The project was started as an experiment by [Nader Dabit](https://twitter.com/dabit3) to see what can be achieved when we bring together talented and likeminded people who are interested in learning about, and building on, Web3.

Reviewer: ```suggestion
The project was started as an experiment by [Nader Dabit](https://twitter.com/dabit3) to see what can be achieved when we bring together talented and like-minded people who are interested in learning about and building on Web3.
```

> As stated on : "Developers around the world are tired of working and contributing their time and effort to enrich the top 1%. Join the movement that is community owned, building the future from the bottom up."

Reviewer: Where was this stated?

Author: This was stated in the contract code. Mentioned in #11 that reading it back didn't really fit so happy to remove. Sure we can work on the text later.

## How do I join?

> The DAO operates inside our [discord sever](https://twitter.com/dabit3). To gain access to this server you need to own a [Devs for Revolution](https://opensea.io/collection/devs-for-revolution) NFT. You can find the contract for purchasing one [here](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b) and a good how to article on how to purchase one from @jkup, [here](https://jonkuperman.com/how-to-join-developer-dao/ ). 

Reviewer: ```suggestion
The DAO operates inside our [discord sever](https://twitter.com/dabit3). To gain access to this server you need to own a [Devs for Revolution](https://opensea.io/collection/devs-for-revolution) NFT. The contract for purchasing one [can be found on etherscan.io](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b). If you're not sure how to purchase an NFT, [check out this how-to article](https://jonkuperman.com/how-to-join-developer-dao/) by @jkup. 
```

Should we refer to it as minting rather than purchasing? Not sure what the exact terminology is here.

Reviewer: ```suggestion
The DAO operates inside our [Discord]. To gain access to this server you need to own a [Devs for Revolution](https://opensea.io/collection/devs-for-revolution) NFT. You can find the contract for purchasing one [here](https://etherscan.io/token/0x25ed58c027921e14d86380ea2646e3a1b5c55a8b) and a good how to article on how to purchase one from @jkup, [here](https://jonkuperman.com/how-to-join-developer-dao/ ). 
```

Whilst there is no charge for the NFT you will have to pay a "gas fee" for the Ethereum Blockchain network in order to purchase the NFT. This fee can range wildy from about $40 up to and over $400 at peak times. If the fee is high you may want to wait and come back later. 

> That said, ther is a limited supply of NFT's that can be created.

Reviewer 3: ```suggestion
That said, there is a limited supply of NFT's that can be created.
```

> That said, there is a limited supply of NFT's that can be created.

Reviewer: This is probably a great place to link to one of those tools that shows the remaining available NFTs. Not sure which one we should link to though, I think I've seen a few.

Reviewer 2: > This is probably a great place to link to one of those tools that shows the remaining available NFTs. Not sure which one we should link to though, I think I've seen a few.

This one helped me the most:  http://ddao.ibby.dev/

Author: Add 👍🏻

> If you get stuck please reach out on our [discord](https://discord.gg/ZHZup79J) and we'd be happy to help.

Reviewer: ```suggestion
If you get stuck please reach out on our [Discord] and we'd be happy to help.
```

## What is this repo?

> This is the code DeveloperDAO's public facing - https://www.developerdoa.com.

Reviewer: ```suggestion
This is the code for the [public DeveloperDAO site](https://developerdao.com/).
```

I think this reads a little better. URL had a small typo too

> At the moment it allows members to view their genesis NFT by entering in it's ID. As the community takes shape we will decide what this website becomes and build it as a collective.

Reviewer: ```suggestion
At the moment it allows members to view their genesis NFT by entering in its ID. As the community takes shape we will decide what this website becomes and build it as a collective.
```

## Built with

- [React.js](https://reactjs.org/)
- [Create React App](https://reactjs.org/docs/create-a-new-react-app.html)
- [TailwindCSS](https://tailwindcss.com/)
- [Yarn](https://yarnpkg.com/)
- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)
- [Prettier](https://prettier.io/)
- [ESLint](https://eslint.org/)
- [Husky](https://typicode.github.io/husky/#/)

## Developing

### Local Development Environment

> 1. Clonse this repo with git

Reviewer 3: ```suggestion
1. Clone this repo with git
```

> 2. install dependencies by running `yarn` within the directory you clone the project (probably `developer-dao`)

Reviewer 3: We can safely omit the rest as it's kinda implied
```suggestion
2. Install the dependencies by running `yarn` 
```

3. Start the development server with `yarn start`
4. Open development site by going to `http:localhost:3000`

## Contributing

Thanks for showing interest in contributing to DeveloperDAO. Before submitting any changes please review our contributing gudielines in [CONTRIBUTING.md].

## Authors

<a href="https://github.com/developer-dao/developer-dao/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=developer-dao/developer-dao" />
</a>

> 

Reviewer: ```suggestion

[Discord]: https://discord.gg/ZHZup79J
```

Allows us to define the link in a single place and reference it throughout the doc. If we change it in the future, we only need to change this one!
